src/client/app/containers/editor.mode.ts

```diff
@@ -264,4 +280,50 @@ export class EditorMode {
     noop() {
         // no-op
     }
+
+    checkForEditorTryIt() {
+        if (!this.isEditorTryIt) {
+            return;
+        }
+
+
+        // Mimic view mode code, except Script Lab contains all editor features
+        let sub = this._route.params
+            .map(params => ({ type: params.type, host: params.host, id: params.id }))
+            .mergeMap(({ type, host, id }) => {
+                // Default to Excel
+                if (!host) {
```

Comment:
Is this necessary? Do we include host in the tryit page url?

Reply:
Not necessary, since the try it page will have the appropriate iframe url